to_json.wdl

```diff
@@ -0,0 +1,237 @@
+			rc_pd = pd.read_json(rc_json)
+			gff_rc_pd = pd.merge(gff_pd, rc_pd, on = ["id", "seqid", "featuretype", "strand", "length"])
+
+			cds_only = gff_rc_pd[gff_rc_pd['featuretype'] == "CDS"]
+
+			sense_reads = cds_only[cds_only['strand'] == "+"].drop(columns = ["antisense_read_count", 
+												"meanA",
+												"medianA",
+												"stdevA"])
+			antisense_reads = cds_only[cds_only['strand'] == "-"].drop(columns = ["sense_read_count", 
+												"mean",
+												"median",
+												"stdev"])
+
+			sorted_features = gff_rc_pd.sort_values(by='sense_read_count', ascending=False)
+			top100 = sorted_features[:100]
+			
+			write_json(gff_rc_pd.to_dict(orient="records"), gff_rc_json)
+			write_json(cds_only.to_dict(orient="records"), cds_json)
+			write_json(sense_reads.to_dict(orient="records"), sense_json)
+			write_json(antisense_reads.to_dict(orient="records"), anti_json)
+			write_json(sorted_features.to_dict(orient="records"), sorted_json)
+			write_json(top100.to_dict(orient="records"), top100_json)
+			
+			sorted_features.to_csv(sorted_tsv, sep="\t") 
+			
+			print("Additional JSON files and tables printed.")
+
+			
+		def write_json(js_data, file_out: str):
+			with open(file_out, 'w') as json_file:
+				json.dump(js_data, json_file, indent=4)
+
+		# Classes #######################################################################################################################################
+		class GTFtoJSON():
+			"""
+			Converts GTF files to JSON records.
+
+			Utilizes package gffutils to create database from gff / gtf files in gtf_json()
+			Extracts desired attributes and features to json using collect_features().
+			Utilizes package json to write json out and package os to check for db existence. 
+			"""
+
+			def __init__(self, gtf_file_name: str, out_json_file: str):
+				"""
+				gtf_file_name: string of gtf or gff file, relative or absolute path both work
+				out_json_file: name of desired json output file, relative or absolute
+				"""
+				self.gtf_file_name = gtf_file_name
+				self.out_json_file = out_json_file
+
+			def gtf_json(self):
+				"""
+				A function that converts a gff file to JSON file.
+				Reads gff in and exports to db (SQL) type file.
+				Uses db type format to channel to collect_features and extract attributes to dictionary before writing to json.
+				"""
+				# read in the gff file to a database
+				if os.path.exists("metat_db.db") is False:
+					gtf_as_db = gffutils.create_db(self.gtf_file_name, dbfn="metat_db.db", force=True,
+												keep_order=True,
+												merge_strategy="create_unique")
+					print("New gffutils db created")
+				else:
+					gtf_as_db = gffutils.FeatureDB("metat_db.db", keep_order=True)
+					print("Cached gffutils db loaded")
+				json_list = []
+				for feat_obj in gtf_as_db.all_features():
+					feat_dic = {}  # an empty dictionary to append features
+					feat_dic_str = self.collect_features(feat_obj=feat_obj, feat_dic=feat_dic)
```

Comment:
I think it is not recommended to pass an empty dict as input to a function: https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python